WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderCardUiState.kt

```diff
@@ -84,12 +86,20 @@ sealed class ReaderCardUiState {
             val name: String,
             val url: String,
             val blogId: Long,
-            val feedId: Long?,
+            val feedId: Long,
             val description: String,
             val iconUrl: String?,
-            val onItemClicked: (Long, Long?) -> Unit
+            val isFollowed: Boolean,
+            val onItemClicked: (Long, Long) -> Unit,
+            val onFollowClicked: (ReaderRecommendedBlogUiState) -> Unit
         ) {
             val isDescriptionVisible: Boolean = description.isNotEmpty()
```

Comment:
We usually make the description nullable and when it's null we hide the view -> there is a helper method in UiHelpers.setTextOrHide. We might want to consider using it. Wdyt?

Reply:
hmm yeah, I think I would prefer to have description nullable instead of an empty string and handle this when building the UI model. I did that as I kind of stuck to the way API works (it sends empty description instead of null)

Reply:
We might want to consider transforming the empty string to null in the VM layer. Wdyt?

Reply:
Yeah I agree :) That's what I tried to say in the previous comment :D

Reply:
Ohh I see :D I misread your previous comment. My bad.

Reply:
I unresolved it to not forget to fix it :)